MIGRATION.md

```diff
@@ -0,0 +1,67 @@
+# Migrating from JSONForms 1 (AngluarJS)
+The complexity of the migration of an existing JSONForms 1 application, which is based on AngularJS, to JSONForms 2 depends on the feature set you use.
+
+## Main Features of JSONForms 2
+There are two big changes between JSONForms 1 and JSONForms 2 you need to understand when migrating your existing application.
+
+1. JSONForms 2 uses [redux](https://redux.js.org/) in order to maintain the state. Your application will need to add redux as a dependency.
```

Comment:
Rephrase: Since JSON Forms 2 maintains its internal state via redux, you will need to add it as a dependency to your application.